common/libs/framing/serial_listeners.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __SERIAL_LISTENERS_H
+#define __SERIAL_LISTENERS_H
+
+enum RxError_t {
```

Comment:
Yes, I restrained myself from renaming stuff in DMA_UART so that changes there are more easily visible. I am planning to do renaming there after this PR is done. Also I could do it now, but that will add more noise to diffs.